spec/features/rooms/index_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'rails_helper'
+
+RSpec.describe "Rooms Index Page" do
+  before(:each) do 
+    @guest_1 = Guest.create!(name: "Kimberly Kimbers", nights: 2)
+    @guest_2 = Guest.create!(name: "Timothy Timmons", nights: 7)
+    @guest_3 = Guest.create!(name: "Nathan Nathaniels", nights: 5)
+
+    @hotel_1 = Hotel.create!(name: "Fancy Hotel", location: "Los Angeles") 
+    @hotel_2 = Hotel.create!(name: "Subpar Hotel", location: "Denver")
+
+    @room_1 = @hotel_1.rooms.create!(rate: 300, suite: "Standard")
+    @room_2 = @hotel_1.rooms.create!(rate: 450, suite: "Penthouse")
+    @room_5 = @hotel_1.rooms.create!(rate: 500, suite: "Presidential")
+
+    @room_3 = @hotel_2.rooms.create!(rate: 50, suite: "Standard")
+    @room_4 = @hotel_2.rooms.create!(rate: 75, suite: "Presidential")
+
+    @guest_rooms_1 = GuestRoom.create!(guest_id: @guest_1.id, room_id: @room_1.id)
+    @guest_rooms_2 = GuestRoom.create!(guest_id: @guest_1.id, room_id: @room_2.id)
+
+    @guest_rooms_3 = GuestRoom.create!(guest_id: @guest_2.id, room_id: @room_1.id)
+
+    @guest_rooms_4 = GuestRoom.create!(guest_id: @guest_3.id, room_id: @room_3.id)
+    @guest_rooms_5 = GuestRoom.create!(guest_id: @guest_3.id, room_id: @room_4.id)
+  end
+
+  describe 'When visiting the rooms index page' do
+    it 'has a list of all the rooms including (suite, rate, name of hotel, number of guests)' do 
+      visit "/rooms"
+
+      within("##{@room_1.id}") do 
```

Comment:
Your tests honestly look super organized and well-written. One bit of advice I have for you when using IDs in HTML is to preface them with some text, and have the ID either in the middle or at the end of the ID string itself. 

This ID is technically something like `#3`; potentially lots of things could have the same ID on a page. A better ID might be `#room_3`. This way you know what kind of object the 3 applies to.